_src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.css

```diff
@@ -230,6 +231,67 @@
   margin: 0 10px;
 }
 
+.b-productswithinputdevices-container.second-template {
+  padding: 0;
+  margin: 0;
+}
+.b-productswithinputdevices-container.second-template .b-productswithinputdevices {
+  padding: 0;
+  margin: 0;
+  max-width: 300px;
+}
+.b-productswithinputdevices-container.second-template .b-productswithinputdevices > div:nth-child(2) {
+  margin-top: 0;
+}
+.b-productswithinputdevices-container.second-template .b-productswithinputdevices > div:nth-child(2) > div {
+  margin: 0;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  width: 300px;
+}
+.b-productswithinputdevices-container.second-template .b-productswithinputdevices > :first-child {
+  padding: 14px;
+}
+.b-productswithinputdevices-container.second-template .b-productswithinputdevices > :first-child b {
+  font: normal normal bold 18px/21px Roboto, sans-serif;
+  color: #000;
+  border-bottom: 1px solid #CCC;
+  margin-bottom: 0.8em;
+  padding-bottom: 0.6em;
+  display: block;
+  width: 92%;
+  margin: 0 auto;
+}
+.b-productswithinputdevices-container.second-template .b-productswithinputdevices > div > div p:nth-child(1) strong {
```

Comment:
is this 100% always strong ? if this strong is part of content, then we shouldn't rely on this